src/main.ts

```diff
@@ -54,6 +64,7 @@ async function getDiff(
     pull_number,
     mediaType: { format: "diff" },
   });
+  console.log(">>> diff response", response)
```

Comment:
Remove the `console.log` statement as it may expose sensitive information in production.